web/app/themes/ppj/partials/mobileNavigation.php

```diff
@@ -32,7 +32,13 @@
 <div class="mobile-nav">
     <div class="mobile-nav__overlay"></div>
     <button class="mobile-nav__close-button"
+            aria-label="Close mobile mobile navigation menu button"
```

Comment:
I think this could be a bit wordy. Would `aria-label="Close Menu"` be better?

Either way, there's a typo – you've repeated "mobile" twice. I also think including the word "mobile" here adds no value, since to the user that's the only navigation menu available to them. Whilst it makes sense to us as developers, I don't think the "mobile" nav needs to be labelled as such to end users.

Reply:
Duplicates removed. Will discuss text with Josh.